### PR TITLE
Minor MapBlock and RowBlock improvements

### PR DESCRIPTION
**Pull request by dain**

### CODE REVIEW

File: presto-spi/src/main/java/com/facebook/presto/spi/block/RowBlock.java [hunk 1]
```diff
@@ -37,20 +37,18 @@
     private final long retainedSizeInBytes;
 
     /**
-     * Create an row block directly from columnar nulls, field blocks, and offsets into the fields.
+     * Create an row block directly from columnar nulls, and field blocks.
      * A null row must have no entries.
```
Comment (nezihyigitbasi) [inline presto-spi/src/main/java/com/facebook/presto/spi/block/RowBlock.java]: This javadoc line was probably referring to the check L48, now that check is removed do we still want to keep this line?

File: presto-spi/src/main/java/com/facebook/presto/spi/block/RowBlock.java [hunk 2]
```diff
@@ -37,20 +37,18 @@
     private final long retainedSizeInBytes;
 
     /**
-     * Create an row block directly from columnar nulls, field blocks, and offsets into the fields.
+     * Create an row block directly from columnar nulls, and field blocks.
```
Comment (nezihyigitbasi) [inline presto-spi/src/main/java/com/facebook/presto/spi/block/RowBlock.java]: `an row block` -> `a row block`

Comment (nezihyigitbasi) [inline presto-spi/src/main/java/com/facebook/presto/spi/block/RowBlock.java]: `,` is unnecessary.

File: presto-spi/src/main/java/com/facebook/presto/spi/block/MapBlock.java [hunk 3]
```diff
@@ -44,6 +44,13 @@
     private volatile long sizeInBytes;
     private final long retainedSizeInBytes;
 
+    /**
+     * Create an map block directly from columnar nulls, keys, values, and offsets into the keys and values.
```
Comment (nezihyigitbasi) [inline presto-spi/src/main/java/com/facebook/presto/spi/block/MapBlock.java]: `an` -> `a`

File: presto-spi/src/main/java/com/facebook/presto/spi/block/MapBlock.java [hunk 4]
```diff
@@ -44,6 +44,13 @@
...
+     * A null map must have no entries.
+     * @param mapType Type of Map; signature is Map(K, V)
```
Comment (nezihyigitbasi) [inline presto-spi/src/main/java/com/facebook/presto/spi/block/MapBlock.java]: `Type of Map` -> `type of map` (the other params start with lowercase)

File: presto-spi/src/main/java/com/facebook/presto/spi/block/MapBlock.java [hunk 5]
```diff
@@ -87,6 +94,9 @@ public static MapBlock fromKeyValueBlock(
 
     /**
      * Create a map block directly without per element validations.
+     * @param keyType Key type K
```
Comment (nezihyigitbasi) [inline presto-spi/src/main/java/com/facebook/presto/spi/block/MapBlock.java]: `key type K` (the other params start with lowercase)

File: presto-spi/src/main/java/com/facebook/presto/spi/block/MapBlock.java (hunk 4 above)
Comment (findepi) [inline presto-spi/src/main/java/com/facebook/presto/spi/block/MapBlock.java]: Intellij formatter insist on having an empty line before first `@param`

**Pull request closed without merging**